scripts/lib/api/updateLinks.ts

```diff
@@ -25,6 +25,26 @@ import { removePart, removePrefix } from "../stringUtils";
 import { HtmlToMdResultWithUrl } from "./HtmlToMdResult";
 import { remarkStringifyOptions } from "./commonParserConfig";
 import { Link } from "../sharedTypes";
+import { ObjectsInv } from "./objectsInv";
+import { transformSpecialCaseUrl } from "./specialCaseResults";
+
+/**
+ * Anchors generated from markdown headings are lower case
+ * As a heuristic, we assume urls containing periods are anchors to HTML id
+ * tags, and anchors with no periods are from markdown headings. This seems to
+ * work ok.
+ */
+function ConvertMarkdownAnchorsToLowerCase(url: string): string {
```

Comment:
```suggestion
function lowerCaseIfMarkdownAnchor(url: string): string {
```

Reply:
Resolved in 79de476953612d192742ffbfad62b075a94f2191